### DISCUSSION TITLE
`itemize` autocompletion does not work with `beamer` class

### DISCUSSION THREAD

**Discussion by K-Maehashi** [James-Yu/LaTeX-Workshop#3114] [Q&A]
## Preliminary questions [Required]
If I set the document class to `article` (or `letter`), the autocompletion for `itemize` works but not with `beamer`. How should I set it up? I checked [this section](https://github.com/James-Yu/LaTeX-Workshop/wiki/Intellisense#commands) and `
LaTeX-Workshop/data/packages/` folder but could not figure out by myself.

### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes  
The only issue I could find on autocompletion was "Autocompletion for \includegraphics seems incomplete".

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]
`itemize` autocompletion does not work.

### To Reproduce
Type `\begin{itemi`

### Expected behavior

Autocompletion for `itemize` shows up.

## Screenshots

![Screen Shot 2021-06-03 at 12 19 49](https://user-images.githubusercontent.com/28519725/120678730-68ddb280-c466-11eb-9f19-93193c491b0f.png)
![Screen Shot 2021-06-03 at 12 20 03](https://user-images.githubusercontent.com/28519725/120678732-69764900-c466-11eb-8f69-559369f349cc.png)
![Screen Shot 2021-06-03 at 12 20 50](https://user-images.githubusercontent.com/28519725/120678734-69764900-c466-11eb-8877-1050aaf1ddda.png)

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: macOS Mojave
 - VS Code version: Version: 1.56.2 
 - Extension version: v8.18.0
 - TeX distribution version: TeX Live 2018

### Are you using VSCodium?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.
- Please paste a complete sample document. Not screenshots.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
